doc/source/release-notes/Percona-Server-5.7.23-23.rst

```diff
@@ -0,0 +1,80 @@
+.. rn:: 5.7.23-23
+
+========================
+Percona Server |release|
+========================
+
+Percona is glad to announce the release of Percona Server |release| on
+September 3, 2018. Downloads are available `here
+<http://www.percona.com/downloads/Percona-Server-5.7/Percona-Server-5.7.23-23/>`_
+and from the :doc:`Percona Software Repositories </installation>`.
+
+This release is based on `MySQL 5.7.23
+<http://dev.mysql.com/doc/relnotes/mysql/5.7/en/news-5-7-23.html>`_
+and includes all the bug fixes in it. |Percona Server| |release| is
+now the current GA (Generally Available) release in the 5.7 series.
+
+All software developed by Percona is open-source and free.
+
+New Features
+============
+
+* The :variable:`max_binlog_files` variable is deprecated and replaced
+  with with the :variable:`binlog_space_limit` variable. The behavior of
+  :variable:`binlog_space_limit` is consistent with the variable
+  :variable:`relay-log-space-limit` used for relay logs and having the same
+  semantics. For more information, see :psbug:`275`.
+* Starting with |release|, all data are encrypted in the InnoDB system
+  tablespace and in the parallel double write buffer. A new variable
+  :variable:`innodb_sys_tablespace_encrypt` is introduced to encrypt the system
+  tablespace. The encryption of the parallel double write buffer file is
+  controlled by the variable :variable:`innodb_parallel_dblwr_encrypt`. Both
+  variables are ``OFF`` by default. For more information, see :psbug:`3822`.
+* Changing ``rocksdb_update_cf_options`` shows warnings and errors in the
+  client. For more information, see :psbug:`4258`.
```

Comment:
returns any warnings and errors to the client instead of printing them to the server error log